code/controllers/subsystem/shuttle.dm

```diff
@@ -74,12 +71,32 @@ SUBSYSTEM_DEF(shuttle)
 
 	if(!transit_turfs.len)
 		setup_transit_zone()
-	initial_move()
+
+	initial_load()
+
 #ifdef HIGHLIGHT_DYNAMIC_TRANSIT
 	color_space()
 #endif
+
+	if(!arrivals)
+		WARNING("No /obj/docking_port/mobile/arrivals placed on the map!")
+	if(!emergency)
+		WARNING("No /obj/docking_port/mobile/emergency placed on the map!")
+	if(!backup_shuttle)
+		WARNING("No /obj/docking_port/mobile/emergency/backup placed on the map!")
+	if(!supply)
+		WARNING("No /obj/docking_port/mobile/supply placed on the map!")
 	..()
 
+/datum/controller/subsystem/shuttle/proc/initial_load()
+	if(!istype(manipulator))
+		CRASH("No shuttle manipulator found.")
+
+	for(var/s in stationary)
```

Comment:
Since we don't have mobile docks on the maps anymore by default you should put a warning before this if one exists.

Reply:
We do have mobile docks on the maps, I haven't transfered pods, the mining base, the Pubby monestary shuttle, or the Nuke ops assault pod.